tests/hacker.test.js

```diff
@@ -685,6 +686,98 @@ describe("PATCH update one hacker", function() {
         });
     });
 
+    it("should FAIL to accept a hacker on /api/hacker/acceptEmail/:email due to authentication", function(done) {
+        chai.request(server.app)
+            .patch(`/api/hacker/acceptEmail/${TeamHacker0.email}`)
```

Comment:
Believe it should be the account's email as emails are tied to accounts and not hackers.